extensions/juejin/README.md

```diff
@@ -0,0 +1,3 @@
+# 掘金
+
+展示掘金各个分类下的推荐文章
```

Comment:
Same here, if you really wants it, then you can make a README-cn.md or something like that